exercise1/task8/app.js

```diff
@@ -0,0 +1,2 @@
+const a = 'meat'
+if (a==='meat'){console.log('Cat likes meat')}
```

Comment:
In javascript programming language == and === have different meanings read more from: 
https://www.guru99.com/difference-equality-strict-operator-javascript.html